examples/fscloud/variables.tf

```diff
@@ -42,7 +39,7 @@ variable "resource_group" {
 variable "existing_at_instance_crn" {
   type        = string
   description = "Optionally pass an existing activity tracker instance CRN to use in the example. If not passed, a new instance will be provisioned"
-  default     = null
```

Comment:
please add this back. if `null` is passed, the example will create a new AT instance